item-client/src/components/Items/ItemContainer.js

```diff
@@ -48,27 +52,92 @@ const useStyles = makeStyles((theme) => ({
   control: {
     padding: theme.spacing(2),
   },
+  modalStyle: {
+    width: 750,
+    position: "fixed",
+    left: 25,
+    top: 15,
+  },
 }));
 
 export const ItemContainer = () => {
+  const [formDisplay, setFormDisplay] = useState("none");
+  const [open, setOpen] = React.useState(false);
+  const [items, setItems] = useState([]);
+  const [item, setItem] = useState({});
+
   const classes = useStyles();
 
-  const renderItems = () => {
-    return items.map((indItem, idx) => {
-      return (
-        <Grid item>
-          <ItemCard indItem={indItem} />
-        </Grid>
-      );
+  useEffect(() => {
+    axios.get("http://localhost:8080/items").then(
+      (response) => {
+        console.log(response.data);
+        setItems(response.data);
+      },
+      [item]
```

Comment:
I think this [item] is in the wrong spot. Im not sure exactly what you are trying to do with it, but if you are trying to re-render the component and run the useEffect when item changes it should go after the } on line 79